File: testdata/changeSignature/fromScala/AddNewClauseWithDefault_after.scala
class AddNewClauseWithDefault {
  def foo(b: Boolean = true)(x: Int, y: Int = 0) = {}

  foo()(1)
  this.foo()(1)
}

class AddNewClauseWithDefault2 extends AddNewClauseWithDefault {
  override def foo(b: Boolean)(x: Int, y: Int): Unit = {
    super.foo()(x)
  }
}

File: scala/scala-impl/testdata/changeSignature/fromScala/AddNewClauseWithDefault_after.java
public class AddNewClauseWithDefaultJava extends AddNewClauseWithDefault {
    @Override
    public void foo(boolean b, int x, int y) {
        AddNewClauseWithDefault a = new AddNewClauseWithDefault();
        a.foo(a.foo$default$1(), 1, a.foo$default$3(a.foo$default$1()));


        super.foo(foo$default$1(), x, foo$default$3(foo$default$1()));
    }
}
